cluster/gce/config-default.sh

```diff
@@ -26,6 +26,7 @@ MASTER_DISK_SIZE=${MASTER_DISK_SIZE:-20GB}
 MINION_DISK_TYPE=pd-standard
 MINION_DISK_SIZE=${MINION_DISK_SIZE:-100GB}
 REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-true}
+PREEMPTIBLE_MINION=false
```

Comment:
Can you follow the style `PREEMPTIBLE_MINION=${PREEMPTIBLE_MINION:-false}` so that it's easy for folks to change this without needing to actually modify this file. (same comment on config-test.sh)